### PR TITLE
Add qblocker

### PR DESCRIPTION
**Pull request by xiaohanyu**
#### Editing an existing cask
- [ ] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

**Comment by adidalal** [comment]
There was a [PR template](https://github.com/caskroom/homebrew-cask/blob/master/.github/PULL_REQUEST_TEMPLATE.md) when you submitted. It must be filled.

File: Casks/qblocker.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask '1clipboard' do
```
Comment (fanquake) [inline Casks/qblocker.rb]: 1clipboard ?

File: Casks/qblocker.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'https://dl.devmate.com/uk.co.wearecocoon.QBlocker/QBlocker.dmg'
```
Comment (vitorgalvao) [inline Casks/qblocker.rb]: Needs a `# dl.devmate.com/uk.co.wearecocoon.QBlocker was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/qblocker.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Use `https://github.com/steve228uk/QBlocker/releases.atom`.

**Comment by adidalal** [comment]
Many errors with this one, I'll go ahead and make the cask

**Comment by xiaohanyu** [comment]
@adidalal Thanks. First time to contribute to this project, lots of things to be familiar with.

**Pull request closed without merging**